### ISSUE TITLE
СКД: Добавить доступ к настройке расшифровке выбранных полей

### ISSUE THREAD

**Issue by AllexAllex** [SeiOkami/OneS#164]
Добавить в СКД возможность настройки расшифровки для каждого поля без использования макетов. Например это можно было бы реализовать как отдельную вкладку Расшифровки на закладке Настройки в конструкторе схемы.
Решаемая задача:
1. Зачастую для программной обработки требуется задать структуру в каждой ячейке расшифровки поля, сейчас это реализуется через макеты (добавление макета группировки или макета поля, и установки свойства ПараметрРасшифровки в интересующих ячейках, для того что бы при обходе программно считывать значения этих расшифровок.
2. Для полей требуется установить основное действие, например при двойном клике я хочу что бы не открывалось меню Расшифровки, а срабатывало "Открыть значение". Сейчас всегда по умолчанию основное действие "Расшифровать"

**Comment by SeiOkami** [comment]
Если мне не изменяет память, то всё это можно прописать именно в поле макета. Неудобно, но можно. В т.ч. действие при двойном клике. 

Но соглашусь, что расшифровка в СКД - это одно из самых непрозрачных и неудобных решений. Это предложение бы обмозговать

**Comment by SeiOkami** [comment]
Отправлено боту 31.03.2023
В чат: https://t.me/e1c_community/144404

**Comment by tormozit** [comment]
> можно прописать именно в поле макета

Автор же просил "без использования макетов"

**Comment by AllexAllex** [comment]
Да, смысл вынести эти настройки на уровень настройки вариантов отчета... там макета нельзя сделать. Пользователи сами варианты настраивают, работают, а потом через много месяцев хотят изменить действие расшифровки для определенных полей.

**Comment by SeiOkami** [comment]
Да, согласен, отправлено боту
